runtime/src/chpl-init.c

```diff
@@ -51,6 +51,18 @@ chpl_main_argument chpl_gen_main_arg;
 
 char* chpl_executionCommand;
 
+void* chpl_string_literals_buffer;
+
+const char* allocate_string_literals_buf(int64_t s) {
+  chpl_string_literals_buffer = chpl_malloc(s);
```

Comment:
Should this allocation go through `chpl_mem_alloc` etc and have a memory descriptor?

Reply:
Right, I can add a metadata entry, but the string literals buffer is never tracked in regular use, because the string literals are initialized before the config const chpl_memTrack is set.